azure-iot-device/azure/iot/device/common/mqtt_transport.py

```diff
@@ -158,8 +181,12 @@ def connect(self, password=None, client_certificate=None):
         self._mqtt_client.tls_insecure_set(False)
         self._mqtt_client.username_pw_set(username=self._username, password=password)
 
-        self._mqtt_client.connect(host=self._hostname, port=8883)
         self._mqtt_client.loop_start()
```

Comment:
>self._mqtt_client.loop_start() [](start = 8, length = 30)

Also I am not entirely sure but does it matter that now loop start is before the connect ?

Reply:
I believe loop_start is supposed to be after connect, at least, that's how all the samples in the docs have it.

---
In reply to: [302767253](https://github.com/Azure/azure-iot-sdk-python-preview/pull/136#discussion_r302767253) [](ancestors = 302767253)

Reply:
brain fart

---
In reply to: [303132683](https://github.com/Azure/azure-iot-sdk-python-preview/pull/136#discussion_r303132683) [](ancestors = 303132683,302767253)